### ISSUE TITLE
<dt> and <dd> elements should be contained by <dl>

### ISSUE THREAD

**Issue by dbranchini** [sul-dlss/purl#786]
Currently on the PURL page, the metadata is displayed as field name `<dt>` and then value `<dd>` without a containing `<dl>`. It seems that all of the sections EXCEPT the **_description_** section are done correctly. I used this page - https://purl.stanford.edu/qf794pv6287 and  https://purl.stanford.edu/ck726bm9949 in my test. Per WCAG 2.0 A (as well as later versions) guidance, it should be contained by an `<dl>` element, otherwise it is invalid. Per Deque University guidance (Axe), affects blind, deafblind, and user with mobility limitations. More follows:

[Specific Issue Outlined by Axe with highlighted issue](https://axe.deque.com/issues/a83b3fc2-c361-4f0e-bf3a-aae0a6177ecd) 
[More information on required HTML by Axe](https://dequeuniversity.com/rules/axe/4.8/dlitem?application=AxeChrome&__hstc=174910525.f7a26c9854d03ed7144bc2a92dee7257.1698865706807.1699027528691.1699290052270.4&__hssc=174910525.1.1699290052270&__hsfp=3599258620)

**Comment by ndushay** [comment]
This is related to sul-dlss/mods_display/issues/153

**Comment by ndushay** [comment]
current thinking:  we want mods-display to use dl by default for all appropriate <dt><dd> frags, but there should be a way to opt out.

**Comment by lwrubel** [comment]
Changes to resolve this should be made in PURL, and not mods_display, so as not to break any usage in other applications.

**Comment by ndushay** [comment]
This can only happen in purl;  there are only a few spots in purl when we don't have `<dl>` wrapper tags.

We want to make sure that changing mods-display doesn't break anything.   Do consumers of mods-display render `<dl>` themselves?  Or should it be done by mods-display for all apps.

**Comment by thatbudakguy** [comment]
discussion on https://github.com/sul-dlss/mods_display/issues/158 seems to indicate that changing mods_display would break things, because consuming apps need the ability to choose what goes into their own `<dl>` and thus render it themselves.

i think we have to solve this problem in purl only.

**Comment by justinlittman** [comment]
I think this is fixed in the new design. @dbranchini Can you verify?

**Comment by dbranchini** [comment]
Looks good to me! Thank you!
